lira_workflow_transitions_by_task_type/README.rst

```diff
@@ -0,0 +1,36 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+======================================
+Lira Workflow Transitions By Task Type
+======================================
+
+This module integrates module ``project_workflow_transitions_by_task_type`` with module ``lira``.
```

Comment:
With time a more complete README with proper links to Lira project is welcome

Reply:
Sure ....